### DISCUSSION TITLE
Question : why a:not([href]) disappeared ?

### DISCUSSION THREAD

**Discussion by micktaiwan** [twbs/bootstrap#31954] [General] [answered]
I have upgraded from 4.5.0 to 4.5.3 and a:not([href]) definition has disappeared.
Just curious why.

**Comment by florianlacreuse** [comment, marked as answer]
Are you referring to that?https://github.com/twbs/bootstrap/blob/4fc35bdc3c3bfd279d199411080e1063772aa643/scss/_reboot.scss#L202-L210

See #30830 and #30726.
